src/plugins/layout/stateprovider.ts

```diff
@@ -89,12 +89,20 @@ export function createElementLayoutState(element: Element): IElementLayoutState
 
   elementState.layout = null;
   if (rect) {
-    elementState.layout = {
-      x: Math.round(rect.left),
-      y: Math.round(rect.top),
-      width: Math.round(rect.width),
-      height: Math.round(rect.height)
-    };
+    if (styles["visibility"] !== "hidden") {
+      elementState.layout = {
+        x: Math.round(rect.left),
```

Comment:
Why do we round? How often do we miss thing with fractional pixels (on phones, etc.)?

Reply:
We round for pixel level precision to save on bytes that we transfer. We can discuss this offline if we would like to change the concept to expand from integers to floats but for now this is what we have been using.